web/src/app/chat/lib.tsx

```diff
@@ -385,7 +385,9 @@ export function processRawChatHistory(
       message: messageInfo.message,
       type: messageInfo.message_type as "user" | "assistant",
       files: messageInfo.files,
-      alternateAssistantID: Number(messageInfo.alternate_assistant_id),
+      alternateAssistantID: messageInfo.alternate_assistant_id
```

Comment:
this was causing `alternateAssistantID` to be set to 0 when `messageInfo.alternate_assistant_id` was null 😓